readinglist/views/article.py

```diff
@@ -67,6 +67,7 @@ class Options:
 @crud()
 class Article(BaseResource):
     mapping = ArticleSchema()
+    deleted_mark = ('status', 2)
```

Comment:
It is exactly why I don't want deleted items to be mixed with items to prevent user from trying to sort or paginate on them.